fluent-bit/coralogix/README.md

```diff
@@ -8,18 +8,49 @@ The default values we provide can be overriden according to your needs, the defa
 helm show values coralogix-charts-virtual/fluent-bit-coralogix
 ```
 
-## Installation
+## Installation with default/dynamic app_name and sub_system
+Dynamic App_Name and Sub_System means taking the value from the running containers, so it can be 'namespace', 'pod_name', 'container_name'.
+If you need to override the default values, but still use dynamic app_name and sub_name, please follow these installation instructions: 
 The following environment variables can be overriden via the 'set' flag in the upgrade command:
 * app_name
 * sub_system
 * endpoint
 * logLevel
 
 for example:
+```bash
+helm upgrade fluent-bit-coralogix coralogix-charts-virtual/fluent-bit-coralogix \
+  --install \
+  --namespace=<your-namespace> \
+  --create-namespace \
+  --set "fluent-bit.logLevel=<level>" \
+  --set "fluent-bit.app_name=<app_name>" \
+  --set "fluent-bit.sub_system=<sub_system>" \
```

Comment:
This is not clear as I guess subsystem_name should be some label if so we should indicate it